portfolio/src/main/webapp/style.css

```diff
@@ -79,4 +82,45 @@ ul {
 .fa-linkedin, 
 .fa-linkedin:visited {
   color: #0072b1;
+}
+
+/* Comments Section */
+
+form {
+  padding: 5px;
+}
+
+textarea {
```

Comment:
nit: this is up to you but I prefer to use classes and ids instead of elements so that if down the road I decide to restructure the html, the css hopefully does not need to change.